data/mp/multiplay/skirmish/bonecrusher/bonecrusher.js

```diff
@@ -203,14 +203,14 @@ var policy = [];
 var nf = [];
 nf['policy'] = false;
 
-var enemyDist = 0;
+const enemyDist = 0;
```

Comment:
From BoneCrusher error:
info |10:54:24: [callFunction:1183] Uncaught exception calling function "checkAlly": TypeError - "'enemyDist' is read-only"::
Make "const enemyDist" back to "var" pls

Reply:
changed to `let`